OctoKit/Releases.swift

```diff
@@ -49,6 +49,25 @@ public struct Release: Codable {
 // MARK: request
 
 public extension Octokit {
+    /// Fetches the list of releases.
+    /// - Parameters:
+    ///   - session: RequestKitURLSession, defaults to URLSession.shared()
+    ///   - owner: The user or organization that owns the repositories.
+    ///   - repository: The name of the repository.
+    ///   - completion: Callback for the outcome of the fetch.
+    @discardableResult
+    func listReleases(_ session: RequestKitURLSession = URLSession.shared, owner: String, repository: String, completion: @escaping (_ response: Response<[Release]>) -> Void) -> URLSessionDataTaskProtocol? {
+        let router = ReleaseRouter.listReleases(configuration, owner, repository)
+        return router.load(session, dateDecodingStrategy: .formatted(Time.rfc3339DateFormatter), expectedResultType: [Release].self) { releases, error in
```

Comment:
Diffs aren't automatically wrapped everywhere, so please avoid long lines, as it makes the code harder to review on devices without a wide screen. 100-120 characters line limit should be sufficient in most cases.

Reply:
I agree that we should limit the lines to some amount of characters. I don’t know if 100-120 is the correct number but it should be enforced with some swiftformat/swiftlint rules so it applies everywhere for everyone.

I personally wouldn’t leave this as a mandatory change to get this PR accepted.

Reply:
@MaxDesiatov this should do the job: https://realm.github.io/SwiftLint/line_length.html

Reply:
I'm aware of this one, and SwiftFormat recently also gained an ability to fit code into given line length too 🙂